File: Data Structures & Algorithms/first-missing-positive/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    firstMissingPositive(nums) {
        nums.sort();
        let a;
        let seen = new Set(nums);
        for(let i =1;i<=nums.length+1;i++){
            if(!seen.has(i)){
                a=i;
                break
            }
        }
        return a;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for(int i=0;i<nums.length;i++){
            for(int j=1;j<nums.length;j++){
                if(nums[i]==nums[j] && i!=j){
                    return true;
                }
            }
        }
        return false;
    }
}